physionet-django/project/forms.py

```diff
@@ -924,12 +925,25 @@ def __init__(self, *args, **kwargs):
             access_policy=self.access_policy
         )
 
-        if self.access_policy not in {AccessPolicy.CREDENTIALED, AccessPolicy.CONTRIBUTOR_REVIEW}:
+        if self.access_policy in {AccessPolicy.OPEN, AccessPolicy.RESTRICTED}:
```

Comment:
The logic here could be clearer. It's unclear how the current set of statements are organised. My preference would be a set of if else statements, working through each access policy, e.g.

if open: do x
elif restricted: do y
etc